### PR TITLE
Feature/#93 sanctions list checking

### PR DESCRIPTION
**Pull request by elyk**
This PR covers the following:

- ETL logic for pulling in sanctions list data from: 'https://scsanctions.un.org/resources/xml/en/consolidated.xml'
- Logic for checking to see if there are any matches amongst users, officials, authorized officers, org heads, or partners legal name within a Partner
- Email to send out when matches + admin interface for handling matches

The above are all run by a management command. These should eventually be triggered by some period scan by a cron.

### CODE REVIEW

File: backend/unpp_api/apps/sanctionslist/scans.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                                              partner)
+
+        create_sanctions_match(matched_names, partner, SANCTION_MATCH_TYPES.organization,
+                               "Match found on the partners organization legal name")
+
+
+        # TODO - come back and changes all first/last to full name once changes are in
+
+        # Partner Heads
+        if partner.org_head:
```
Comment (lkolacz) [inline backend/unpp_api/apps/sanctionslist/scans.py]: here we should "TODO" property that is under code review stage...
```
 +    @property
 +    def head_of_organization(self):
 +        if self.is_hq:
 +            return self.org_head
 +        else:
 +            return self.hq.org_head
```

File: backend/unpp_api/apps/sanctionslist/tests.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (lkolacz) [inline backend/unpp_api/apps/sanctionslist/tests.py]: here is a lot of logic that should have cover unittest for this

File: backend/unpp_api/apps/sanctionslist/etl.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+import requests
+import xmltodict
+from dateutil import parser
+
+from common.consts import SANCTION_LIST_TYPES
+from .models import SanctionedItem, SanctionedName
+
+SANCTIONS_LIST_URL = 'https://scsanctions.un.org/resources/xml/en/consolidated.xml'
+ACCEPTED_QUALITIES = ['Good', 'a.k.a.']
+
```
Comment (lkolacz) [inline backend/unpp_api/apps/sanctionslist/etl.py]: those 2 def `SANCTIONS_LIST_URL, ACCEPTED_QUALITIES` IMO should be defined in one place => settings ...
I'm suggesting to replace that...

Comment (elyk) [reply backend/unpp_api/apps/sanctionslist/etl.py]: They won't change, so hence the reason for putting it here. However, I can see reason for SANTIONS_LIST_URL needing to be swapped out for a mock response in tests, so can move that.

File: backend/unpp_api/apps/sanctionslist/scans.py (hunk 1 above)
Comment (elyk) [reply backend/unpp_api/apps/sanctionslist/scans.py]: This isn't really as relevant here. We are looking at each partner individually. What we ultimately do (lock the account) can affect the whole org, but that is a decision later on.

File: backend/unpp_api/apps/sanctionslist/tests.py (hunk 2 above)
Comment (elyk) [reply backend/unpp_api/apps/sanctionslist/tests.py]: Agree. Will have tests in later. There still needs to be additional logic added here for at the point of registration capturing, so will get in then.

**Pull request merged**